### ISSUE TITLE
Where is the API? and many documentation missing

### ISSUE THREAD

**Issue by remybonnav** [jupyter-widgets/ipywidgets#3637]
Hi,

The documentation is missing a lot of important infos.
For example, the "Style" section https://ipywidgets.readthedocs.io/en/8.0.2/examples/Widget%20Styling.html
I could not find what are the values required for each attribute of style (example : what should we enter for description_width?

I could find that style = {'description_width': 'initial'} could be used for example but it was in the section layout.

It would really helpful for beginners to have access to the API because otherwise you mostly have to search on stack overflow for answers.
Is there an API somewhere?
Thanks a lot for your help.

**Comment by jasongrout** [comment]
Good point. We should state in the documentation that any value in the style attributes is a CSS value, and refer people to the browser's CSS documentation. In this case, for a width value, see https://developer.mozilla.org/en-US/docs/Web/CSS/width

**Comment by bollwyvl** [comment]
Have a look here: https://ipywidgets.readthedocs.io/en/latest/reference/index.html
